repo/api_server_repository.go

```diff
@@ -204,8 +208,17 @@ func (r *apiServerRepository) UpdateDescription(d string) {
 }
 
 func (r *apiServerRepository) Close(ctx context.Context) error {
-	if err := r.omgr.Close(); err != nil {
-		return errors.Wrap(err, "error closing object manager")
+	if r.omgr != nil {
```

Comment:
Is `Close()` expected to be called concurrently?

Reply:
Close() is meant to be called without concurrency